x-pack/plugins/security_solution/cypress/e2e/detection_rules/custom_query_rule.cy.ts

```diff
@@ -133,6 +134,7 @@ describe('Custom query rules', () => {
 
       cy.log('Filling define section');
       importSavedQuery(this.timelineId);
+      removeAlertsIndex();
```

Comment:
The behavior of import rule from saved timeline changed with this PR to match the index pattern of the timeline. Because the saved timeline used in tests has `dataViewId: null` and `indexNames: []`, we know that this is a pre-sourcerer timeline when we only used our default security data view, so we treat this timeline to default security data view. Depending on if any alerts have been generated yet, this data view includes `audtibeat-*` and `.alerts-security.alerts-default`. We now need to check if alerts index exists and remove it if it does, or it will break tests.

Reply:
Thanks for the explanation here. I worry that the information you shared here can be forgotten in the future so it won't be so clear why alerts index is cleared here. Is it possible to remove alerts index inside `importSavedQuery()`? If not I'd leave a short comment in the code.

Reply:
yes it looks like this is the only place its used, will do